public/app/plugins/datasource/grafana-azure-monitor-datasource/app_insights/app_insights_datasource.test.ts

```diff
@@ -1,6 +1,6 @@
 import { DataFrame, getFrameDisplayName, toUtc } from '@grafana/data';
 import { TemplateSrv } from 'app/features/templating/template_srv';
-import { backendSrv } from 'app/core/services/backend_srv'; // will use the version in __mocks__
+import { backendSrv } from 'app/core/services/__mocks__/backend_srv'; // ???? will use the version in __mocks__
```

Comment:
any ideas how to make sure it actually uses the mocked version?

Reply:
think have to call jest.mock('app/core/services/backend_srv') if I remember correctly